testsuite/forge.py

```diff
@@ -1044,12 +1044,14 @@ def last(self, limit: int = 1) -> Generator[str, None, None]:
 
 
 def assert_provided_image_tags_has_profile_or_features(
-    forge_image_tag,
-    image_tag,
+    forge_image_tag: Optional[str],
+    image_tag: Optional[str],
     enable_failpoints_feature: bool = False,
```

Comment:
are we ever not passing these in? Where are we calling this without these args?

Reply:
they're just None sometimes from `envoption`

Reply:
I meant the enable_failpoints_feature optional keyword args